File: Frontend/src/models/ChatInfo.ts
import { Friends } from "./Friends";

export interface RecentChatListInfo {
  channelId: string;
  channelName: string;
  channelType: string;
  channelCreatedTime: Date;
  chatInfo: ChatInfo;
}

interface ChatInfo {
  message: string;
  messageTime: Date;
  sendTo: Friends;
  sendBy: Friends;
  messageId: string;
}

File: Backend/Kinder-Backend/Models/ChatInfo.cs
using System;
using Kinder_Backend.Controllers;

namespace Kinder_Backend.Models;

public class ChatInfo
{
    public string Message { get; set; }
    public DateTime MessageTime { get; set; }
    public UserInfo SendTo { get; set; }
    public UserInfo SendBy { get; set; }
    public string MessageId { get; set; }
}